web/assets/js/main_out.js

```diff
@@ -21,22 +23,34 @@
         if (i !== -1) this.splice(i, 1);
         return i !== -1;
     };
-    function bytesToColor(r, g, b) {
-        var r1 = ("00" + (~~r).toString(16)).slice(-2);
-        var g1 = ("00" + (~~g).toString(16)).slice(-2);
-        var b1 = ("00" + (~~b).toString(16)).slice(-2);
-        return `#${r1}${g1}${b1}`;
+    Element.prototype.hide = function() {
+        this.style.display = 'none';
+        this.style.opacity = 0;
+    };
+    Element.prototype.show = function() {
+        this.style.display = 'block';
```

Comment:
Not all elements use block as default, making 'block' into '' will make it use the default one or the one provided by the css files(s)

Reply:
That's a really good solution!

Reply:
It would be a good solution, if it worked.... but it doesn't.